Player 2 player layout

eugene195 opened the pull request.

eugene195 committed:
#2 Created initial player fragment

eugene195 committed:
#2 Moved drawables to project

eugene195 committed:
#2 Fixed fragment fullscreen display

eugene195 committed:
#2 Fragment UI changes

Custom header colors
Circular UI view

eugene195 commented:
Мб выкинуть код, который отображает фрагмент из мейнактивити?

xammi commented on app/src/main/java/ru/bmstu/wundermusik/MainActivity.java:
```diff
@@ -9,44 +10,36 @@
-public class MainActivity extends AppCompatActivity {
+import ru.bmstu.wundermusik.fragments.PlayerFragment;
+
+public class MainActivity extends AppCompatActivity implements PlayerFragment.PlayerActionsListener {
```
Думаю здесь нужно создать новое активити, чисто для плеера

eugene195 replied:
Пофиксил

xammi commented on build.gradle:
```diff
@@ -15,6 +15,7 @@ buildscript {
 allprojects {
     repositories {
         jcenter()
+        maven { url "https://jitpack.io" }
```
Для чего это?

eugene195 replied:
Мавеновская репка лейаута плеера

xammi commented on app/src/main/res/drawable/shadow.xml:
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Ты положил две картинки кита - так нужно?

eugene195 replied:
Пофиксил

eugene195 committed:
#2 Дозработка фрагмента плеера

Создал отдельное активити плеера

eugene195 committed:
#2 Дозработка фрагмента плеера

- убрал лишнюю картинку
- именовал все константы
- подвинул maskProgress до идельных размеров (360)

xammi commented:
# При переворачивании экрана в альбомную ориентацию пользоваться плеером становится невозможно - кит становится маленьким, полоса навигации скрывается

eugene195 committed:
#2 Поправил горизонтальное отображение плеера

eugene195 committed:
#2 Включил тестовые сущности на основной лейаут
